packages/web/components/Landing/OnboardingGuidance/index.tsx

```diff
@@ -0,0 +1,149 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  MetaButton,
+  MetaTile,
+  MetaTileBody,
+  MetaTileHeader,
+  SimpleGrid,
+  Text,
+  VStack,
+} from '@metafam/ds';
+import BabyOctopus from 'assets/quests/baby_octo.png';
+import Octopus from 'assets/quests/octopus.png';
+import SpaceOctopi from 'assets/quests/space_octo.png';
+import { PageContainer } from 'components/Container';
+import { MarkdownViewer } from 'components/MarkdownViewer';
+import { useRouter } from 'next/router';
+import { useEffect, useRef } from 'react';
+
+import roleJson from './guidance.json';
+import { RolePicture } from './RolePicture';
+
+type GuidanceRole = {
+  role: string;
+  details: string[];
+  image: string;
+};
+
+enum Role {
+  CuriousOcto = 'Curious Octo',
+  EngagedOcto = 'Engaged Octo',
+  SpaceOcto = 'Space Octo',
+}
+
+const OnboardingGuidance: React.FC = () => {
+  const router = useRouter();
+  const guidanceRoles = useRef<GuidanceRole[]>([]);
+  useEffect(() => {
+    guidanceRoles.current = roleJson.roles.map(({ role, list }) => {
+      const guidance: GuidanceRole = { role, details: list, image: '' };
+      if (role === Role.CuriousOcto) guidance.image = BabyOctopus.src;
+      if (role === Role.EngagedOcto) guidance.image = Octopus.src;
+      if (role === Role.SpaceOcto) guidance.image = SpaceOctopi.src;
+      return guidance;
+    });
+  }, []);
+
+  const handleClick = () => {
+    router.push('/profile/setup/name');
+  };
+
+  return (
+    <PageContainer w="100%" mt={24} px={8}>
+      <VStack
+        pos="relative"
+        align="center"
+        justify="center"
+        mb={{ base: '3rem' }}
+        spacing={0}
+        maxW="full"
+      >
+        <Text fontWeight={600} fontSize={{ base: '2xl', lg: '3xl', xl: '4xl' }}>
+          Joining MetaGame
+        </Text>
+        <Text
+          fontWeight={400}
+          fontSize="lg"
+          textAlign={{ base: 'center', lg: 'start' }}
+        >
+          Your journey into MetaGame has just begun. Here's how it works:
+        </Text>
+      </VStack>
+      {!!guidanceRoles.current?.length && (
+        <SimpleGrid
+          columns={[1, null, 2, 3]}
+          spacing={{ base: 20, lg: 8 }}
+          autoRows="minmax(35rem, auto)"
+        >
+          {guidanceRoles.current?.map(({ role, details, image }) => (
+            <Box key={role}>
+              <MetaTile height="full" width="full" noTilt>
+                <MetaTileHeader>
+                  <RolePicture src={image} size="xl" />
```

Comment:
You could use just `SquareImage` from `import { SquareImage } from 'components/SquareImage';` here, then you don't need to create the `RolePicture` component.